jsonnet/kube-prometheus/kube-prometheus.libsonnet

```diff
@@ -20,28 +18,33 @@ local blackboxExporter = import './blackbox-exporter/blackbox-exporter.libsonnet
 (import './alerts/alerts.libsonnet') +
 (import './rules/rules.libsonnet') +
 {
-  nodeExporter: nodeExporter({
-    namespace: $._config.namespace,
-    version: '1.0.1',
-    image: 'quay.io/prometheus/node-exporter:v1.0.1',
-  }),
   alertmanager: alertmanager({
     name: 'main',
     namespace: $._config.namespace,
     version: '0.21.0',
     image: 'quay.io/prometheus/alertmanager:v0.21.0',
   }),
+  blackboxExporter: blackboxExporter({
```

Comment:
Why is this part of the kube-state-metrics PR, just random cleanup?

Reply:
yeah, just making things to be in alphabetical order